website/content/partials/kubernetes-supported-versions.mdx

```diff
@@ -4,6 +4,7 @@ The following [Kubernetes minor releases][k8s-releases] are currently supported.
 The latest version is tested against each Kubernetes version. It may work with
 other versions of Kubernetes, but those are not supported.
 
+* 1.29
 * 1.28
 * 1.27
 * 1.26
```

Comment:
I think we should probably also drop 1.24 now?

Reply:
Seems reasonable.

Reply:
Dropped in b3a82b06485fd98ea5e34601d1e931c0fa49f515